tests/unit/PartialFileFieldSubmissionTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Firesphere\PartialUserforms\Tests;
+
+use Firesphere\PartialUserforms\Models\PartialFileFieldSubmission;
+use Firesphere\PartialUserforms\Models\PartialFormSubmission;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\DefaultAdminService;
+use SilverStripe\Security\Security;
+use SilverStripe\UserForms\Model\UserDefinedForm;
+
+class PartialFileFieldSubmissionTest extends SapphireTest
+{
+
```

Comment:
Can you please add `protected $usesDatabase = true;` so it uses tempdb and doesn't insert test data into the local database?